webapp/channels/src/components/system_notice/system_notice.tsx

```diff
@@ -155,7 +155,7 @@ export default class SystemNotice extends React.PureComponent<Props> {
                         {notice.allowForget &&
                             <button
                                 id='systemnotice_dontshow'
-                                className='btn'
+                                className='btn btn-tertiary'
```

Comment:
I think we can add the `btn-sm` class to get the smaller version of the button for these notices. Should apply that to all buttons in these notices